src/pad/src/pad.tcl

```diff
@@ -138,13 +138,14 @@ sta::define_cmd_args "make_io_sites" {-horizontal_site site \
                                       [-rotation_horizontal rotation] \
                                       [-rotation_vertical rotation] \
                                       [-rotation_corner rotation] \
-                                      [-ring_index index]}
+                                      [-ring_index index]
+};# checker off
 
 proc make_io_sites {args} {
   sta::parse_key_args "make_io_sites" args \
     keys {-horizontal_site -vertical_site -corner_site -offset -rotation \
       -rotation_horizontal -rotation_vertical -rotation_corner -ring_index} \
-    flags {}
+    flags {}; # checker off
```

Comment:
why off?

Reply:
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/39641663/c06b0921-19c3-4749-a8b4-994f0aec8f09)

Deprecated key